Code/C++/matrix_operations.cpp

```diff
@@ -0,0 +1,211 @@
+/* C++ CODE TO SHOW DIFFERENT OPERATIONS ON 2D ARRAY LIKE SUBTRACTION, ADDITION, MULTIPLICATION AND FINDING THE INVERSE OF A MATRIX */
+
```

Comment:
Give a description of your code

Reply:
Added sir, please check sir if it's ok now.